### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by hartwork**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,19 +7,19 @@
 # Branches of Interest
 
 - [`0.4.x`](https://github.com/Libvisual/libvisual/tree/0.4.x) —
-  targetting version `0.4.x`,
+  targetting version 0.4.x,
   maintenance,
```
Comment (kaixiong) [inline README.md:12]: Oops, 'targeting' is misspelled (1 t only).

File: README.md [hunk 2]
```diff
@@ -7,19 +7,19 @@
...
   bugfixes only
 
 - [`master`](https://github.com/Libvisual/libvisual/tree/master) —
-  targetting version `0.5.x` and beyond,
+  targetting version 0.5.x and beyond,
```
Comment (kaixiong) [inline README.md:16]: Here as well, 'targeting'.

File: README.md (hunk 1 above)
Comment (hartwork) [reply README.md:12]: @kaixiong it's not a bug, see https://en.wiktionary.org/wiki/targetting . Now replaced with "targets" to e.g. better fit "uses".

Comment (kaixiong) [reply README.md:12]: @hartwork, I saw that same page before commenting :laughing: Here's an interesting [link](https://english.stackexchange.com/questions/205815/is-targetted-a-standard-british-english-spelling).

Comment (hartwork) [reply README.md:12]: @kaixiong good link, ready to merge?

**Review by kaixiong — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kaixiong) [reply README.md:12]: @hartwork, yup approved.

**Pull request merged**